buildconfig/pygame-stubs/camera.pyi

```diff
@@ -24,4 +24,4 @@ class Camera:
     def get_size(self) -> Tuple[int, int]: ...
     def query_image(self) -> bool: ...
     def get_image(self, surface: Optional[Surface] = None) -> Surface: ...
-    def get_raw(self) -> str: ...
+    def get_raw(self) -> bytes: ...
```

Comment:
Is this str on py2 and bytes on py3?

Reply:
Yes. I was assuming the `.pyi` types only applied to python3 since typing is a new thing, but I can change that if otherwise.

Reply:
Cool. So this needs conditional typing like has been done in some other files.